inspector-axon/src/main/java/io/axoniq/inspector/messaging/RollingCountMeasure.kt

```diff
@@ -16,35 +16,33 @@
 
 package io.axoniq.inspector.messaging
 
-import java.util.concurrent.atomic.AtomicLong
+import java.util.concurrent.ConcurrentHashMap
 
 /**
  * Rolls up for every report, which is sent every ten seconds.
```

Comment:
Doc needs to change to the current solution.

Reply:
I will adjust this.
Writing the javadoc, I found the `pruneData` method unwarranted, as it's only cleaned when the value is read anyway. So, the `count()` method will now clean up the values.